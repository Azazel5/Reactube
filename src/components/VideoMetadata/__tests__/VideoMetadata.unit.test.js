import React from 'react'
import {shallow} from 'enzyme'
import VideoMetadata from '../VideoMetadata'

describe('<VideoMetadata />', () => {
    test('renders <VideoMetadata />', () => {
        const wrapper = shallow(<VideoMetadata />)
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <VideoMetadata /> with viewCount prop', () => {
        const wrapper = shallow(<VideoMetadata viewCount={10000}/>)
        expect(wrapper).toMatchSnapshot()
    })
})