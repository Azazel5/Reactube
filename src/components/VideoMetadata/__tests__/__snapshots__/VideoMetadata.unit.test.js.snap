// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoMetadata /> renders <VideoMetadata /> 1`] = `
<div
  className="video-metadata"
>
  <h3>
    Video Title
  </h3>
  <div
    className="video-stats"
  >
    <span>
      NaN
       views
    </span>
    <div
      className="video-actions"
    >
      <rating
        dislikeCount={100}
        likeCount={1000}
      />
      <Button
        as="button"
        basic={true}
        icon={true}
        labelPosition="left"
      >
        <Icon
          as="i"
          name="share"
        />
        Share
      </Button>
      <Button
        as="button"
        basic={true}
        icon={true}
      >
        <Icon
          as="i"
          name="add circle"
        />
      </Button>
      <Button
        as="button"
        basic={true}
        icon={true}
      >
        <Icon
          as="i"
          name="ellipsis horizontal"
        />
      </Button>
    </div>
  </div>
  <Divider />
</div>
`;

exports[`<VideoMetadata /> renders <VideoMetadata /> with viewCount prop 1`] = `
<div
  className="video-metadata"
>
  <h3>
    Video Title
  </h3>
  <div
    className="video-stats"
  >
    <span>
      10,000
       views
    </span>
    <div
      className="video-actions"
    >
      <rating
        dislikeCount={100}
        likeCount={1000}
      />
      <Button
        as="button"
        basic={true}
        icon={true}
        labelPosition="left"
      >
        <Icon
          as="i"
          name="share"
        />
        Share
      </Button>
      <Button
        as="button"
        basic={true}
        icon={true}
      >
        <Icon
          as="i"
          name="add circle"
        />
      </Button>
      <Button
        as="button"
        basic={true}
        icon={true}
      >
        <Icon
          as="i"
          name="ellipsis horizontal"
        />
      </Button>
    </div>
  </div>
  <Divider />
</div>
`;
